test/parallel/test-worker-execargv.js

```diff
@@ -12,7 +12,7 @@ const decoder = new StringDecoder('utf8');
 // Do not use isMainThread so that this test itself can be run inside a Worker.
 if (!process.env.HAS_STARTED_WORKER) {
   process.env.HAS_STARTED_WORKER = 1;
-  const w = new Worker(__filename, { execArgv: ['--trace-warnings'] });
+  const w = new Worker(__filename, { execArgv: [] });
```

Comment:
This test should still make sure `execArgv` is getting passed to the worker. I suggest using `--no-trace-warnings` or another flag.